### PR TITLE
Fix boolean unions when the messages has a True value

### PR DESCRIPTION
**Pull request by fgarces**
When using unions with booleans, messages fail to decode if they contain a true value. 

@cgrdavies  you are prime

cc/ @tjwp since @jturkel is on vacation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@
 /tmp/
 /gemfiles/*.lock
 /log/*
+
+*.iml
+.idea
```
Comment (cgrdavies) [inline .gitignore:14]: we might want to keep editor-specific stuff out of here

**Review by cgrdavies — APPROVED** [review]
Looks good to me!

**Review by tjwp — APPROVED** [review]
One minor comment, but makes sense to me. Let me know if you'd like me to push a new 3.0 release with this fix.

**Comment by fgarces** [comment]
@tjwp I just merged and it would be great if you could push the release.

**Pull request merged**